resource-managers/kubernetes/core/src/test/scala/org/apache/spark/deploy/k8s/features/PodTemplateConfigMapStepSuite.scala

```diff
@@ -56,8 +60,10 @@ class PodTemplateConfigMapStepSuite extends SparkFunSuite {
 
     assert(configuredPod.pod.getSpec.getVolumes.size() === 1)
     val volume = configuredPod.pod.getSpec.getVolumes.get(0)
+    val generatedResourceName = kubernetesConf.resourceNamePrefix +
+      PodTemplateConfigMapStep.PODSPEC_CONFIGMAP_POSTFIX
```

Comment:
Same as the suggestion above, should use the constant already defined in Constants.scala.